test/cli-utils-chromedriver-manager.spec.js

```diff
@@ -0,0 +1,129 @@
+/*jshint jasmine: true, node: true */
+'use strict';
+
+const mock = require('mock-require');
+
+describe('utils/chromedriver-manager.js', () => {
+
+  function spyOnChromeDriverVersionMatcher() {
+    const spyChromeDriverVersionMatcher = jasmine.createSpyObj(
+      'chromedriver-version-matcher',
+      ['getChromeDriverVersion']
+    );
+    mock('chromedriver-version-matcher', spyChromeDriverVersionMatcher);
```

Comment:
I think you'll need some `mock.stopAll()` in an `afterEach`?

Reply:
A good practice.  I'll add it.